cypress/integration/my-orcid-top-bar.e2e.js

```diff
@@ -0,0 +1,84 @@
+/// <reference types="cypress" />
+import { environment } from '../cypress.env'
+
+const randomUser = require('../helpers/randomUser')
+const runInfo = require('../helpers/runInfo')
+
+Cypress.Commands.add(
+  'dragTo',
+  { prevSubject: 'element' },
+  (subject, droppableSelector) => {
+    const coords = Cypress.$(droppableSelector)[0].getBoundingClientRect()
+    cy.wrap(subject)
+      .trigger('mousedown', { which: 1 })
+      .trigger('mousemove', { clientX: coords.x + 10, clientY: coords.y + 10 })
+      .trigger('mouseup', { force: true })
+  }
+)
+describe('My Orcid top bar' + runInfo(), () => {
+  before(() => {
+    cy.programmaticSignin('testUser')
+  })
+  before(() => {
+    cy.visit(`${environment.baseUrl}/qa/my-orcid`)
+  })
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('XSRF-TOKEN', 'JSESSIONID')
+  })
+  describe('Top bar buttons', () => {
+    it('display `Printable version` button', () => {})
+    it('display `Get a QR code for your ORCID iD` button', () => {})
+    it('display `Display your iD on other sites` button', () => {})
+  })
+  describe('Names' + runInfo(), () => {
+    it('display user published name if exists"', () => {})
```

Comment:
We need to add the implementation of some basic tests at least.
Check out the sidebar tests for some inspiration